style.css

```diff
@@ -0,0 +1,220 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    max-width: 1024px;
+    margin: 0 auto;
+    font-family: 'opensans-regular', sans-serif;
+    font-weight: normal;
+    font-size: 15px;
+    line-height: 30px;
+    color: #838C95;
+    background: #161415;
+}
+
+header {
+    position: relative;
+    height: 600px;
+    min-height: 500px;
+    background: #161415;
+    background-size: cover;
+    overflow: hidden;
+}
+
+article, aside, details, figcaption, figure, footer, header, hgroup, menu, nav, section {
+    display: block;
+}
+
+.social {
+    margin: 24px 0;
+    padding: 0;
+    font-size: 30px;
+    text-shadow: 0px 1px 2px rgba(0, 0, 0, .8);
+    display: inline-block;
+    text-align: center;
+}
+
+.social li {
+    display: inline-block;
+    margin: 0;
+    padding: 0;
+    margin-left: 42px;
+}
+
+a {
+    color: #11ABB0;
+    text-decoration: none;
+}
+
+nav {
+    background-color: #333;
+    font: 12px 'Arial', sans-serif;
+    text-transform: uppercase;
+    letter-spacing: 2.5px;
+    width: 100%;
+    margin: 0 auto;
+    z-index: 100;
+    position: fixed;
+    left: 0;
+    top: 0;
+}
+
+ul#nav {
```

Comment:
Better don't use id for styling. Here using 'ul nav' as tag selector is possible, but using class  '.nav' is still preferable